source_modelling/scripts/plot_rakes.py

```diff
@@ -0,0 +1,75 @@
+"""Plot a sample of rake values across a multi-segment rupture."""
+
+from pathlib import Path
+from typing import Annotated
+
+import typer
+
+from pygmt_helper import plotting
+from source_modelling import srf
+
+
+def plot_rakes(
+    srf_ffp: Annotated[
+        Path, typer.Argument(help="Path to SRF file to plot.", exists=True)
+    ],
+    output_ffp: Annotated[
+        Path, typer.Argument(help="Output plot image.", dir_okay=False)
+    ],
+    dpi: Annotated[
+        float, typer.Option(help="Plot output DPI (higher is better).")
+    ] = 300,
+    title: Annotated[str, typer.Option(help="Plot title to use.")] = "Title",
```

Comment:
Maybe the default title should be None? And then if so just doesnt have a title?

Reply:
Yeah! Good idea